lib/saml/errors.rb

```diff
@@ -11,6 +11,9 @@ class UserAttributeError < StandardError
     MHV_ICN_MISMATCH = { code: '103',
                          tag: :mhv_icn_mismatch,
                          message: 'MHV credential ICN does not match MPI record' }.freeze
+    IDME_UUID_MISSING = { code: '104',
```

Comment:
jw are these codes derived from anything we're getting back from other services or just for internal tracking?

Reply:
these don't map to anything else - they are basically the contract between backend and frontend so frontend knows what error page to display to users.